modules/ocf_stats/manifests/grafana.pp

```diff
@@ -0,0 +1,62 @@
+# grafana config
+class ocf_stats::grafana {
+  class { 'ocf::packages::grafana::apt':
+    stage => first,
+  }
+
+  $canonical_url = $::host_env ? {
+    'dev'  => 'dev-prometheus.ocf.berkeley.edu',
+    'prod' => 'prometheus.ocf.berkeley.edu',
+  }
+
+  class { 'grafana':
+    install_method           => 'repo',
+
+    # Unfortunately, puppet-grafana requires that we specify a version number
+    # here.
+    version                  => '5.2.1',
+    manage_package_repo      => false,
```

Comment:
Is there a reason why the whole apt key and repo need to be specified manually instead of letting the module manage both of them? That might fit the version issue too, although I'm not sure on that.

Reply:
I initially looked into installing grafana from the official Debian repo, but found that it's quite out-of-date and that [updating it with new packaging is a pain](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=835210) due to lack of upstream support.

Reply:
The way the module sets up the repo leads to errors in our setup. I'd be willing to look into this more and/or make a PR on their end to fix it.